docs/source/user-guide/general/drift_guide.rst

```diff
@@ -0,0 +1,292 @@
+   :alt: Different types of drift
+   :align: center
+
+Different types of drift. Adapted from `this source <https://www.win.tue.nl/~mpechen/publications/pubs/Gama_ACMCS_AdaptationCD_accepted.pdf>`_.
+
+
+What Can You Do in Case of Drift?
+---------------------------------
+
+When suspecting drift in your data, you must first understand which type of drift it is. However, this is not a trivial
+question:
+
+* If your drifted data has labels, you can easily know which type of drift you are facing.
+* However, the more common case is when you don't have your data labels yet. In this case, we can only try and measure
+  drift on the data (features) or the model's output.
+  If we detect drift, we have no way to know whether this drift is a simple data drift, label drift or concept drift, as
+  we don't know the true labels.
+
+In deepchecks, we show you a drift score and the drift on your most :doc:`important features </user-guide/tabular/feature_importance.rst>`,
+giving you an insight on the severity of your drift, even if your not still sure of its source. Now you can implement one of the following solutions:
+
+Retrain Your Model
+^^^^^^^^^^^^^^^^^^
+
+If you have either kind of drift, retraining your model on new data that better represents the current distribution,
+is the most straight-forward solution.
+However, this solution may require additional resources such as manual labeling of new data, or might not be possible
+if labels on the newer data are not available yet.
+
+Retraining is usually necessary in cases of concept drift. However, retraining may still be of use even for label drift,
+as the model may perform better when knowing the correct distribution of the label (this is not relevant when the
+training dataset is sampled so labels are evenly distributed)
+
+Adjust Your Prediction
+^^^^^^^^^^^^^^^^^^^^^^
+
+When retraining is not an option, or if a quick action needs to be taken, adjustments to the output of the models may
+still help in cases of concept drift. This can be done by either recalibrating your model's output, or by changing your
+decision thresholds on the model's scores.
+
+However, these methods assume that there's still enough similarity between your training data and your current data,
+which may not always be the case.
+
+Do Nothing
+^^^^^^^^^^
+
+Not all drift is necessarily bad, and each case should be examined separately. Sometimes, data drift may be simply
+explained by label drift (for example, in a dataset of food images, a drift in brightness of images can simply mean
+that people are eating more eggs, which are whiter than other foods).
+
+
+How Do You Detect Drift?
+=========================
+
+In general, drift detection is done by comparing the newer and older data, and seeing whether they are derived from
+the same underlying distribution. This is actually not a "yes or no" question - Instead, we ask "how much are these
+distributions different?".
```

Comment:
We don't always compare to current - in research, we hardly ever do.

Reply:
in current data I mean "the current dataset we're comparing to", agree in your comment that it may not be clear, I'm goo also with other phrasings that drive that idea through. Would want it to be clear that we're comparing distributions between two datasets that we care about understanding if there is a difference between them... but it's also not necessarily "newer" and "older". It can also be two datasets collected from different sources. Or that we split a dataset into two and used it for training two different models. Or any other scenario. (summing up - just felt that the "newer" and "older" is limiting)

Reply:
@shir22 I understand where you're coming from, but I disagree. By definition, we're discussing drift over time and not "measured differences between datasets", and because of that, "newer" and "older" are accurate and describe the discussed scenario.